onyo/lib/onyo.py

```diff
@@ -159,7 +161,10 @@ def generate_commit_message(self, message: str = "", cmd: str = "",
             if len(msg) < MAX_LEN:
                 message_header = msg
 
-        return message_header
+        # add list of changed files in alphabetical order to message body
```

Comment:
The "alphabetizing" is done elsewhere, so it feels out of place to specify it here.